examples/morpheusvm/genesis/rules.go

```diff
@@ -18,6 +22,10 @@ type Rules struct {
 	chainID   ids.ID
 }
 
+type FeeMetadata struct {
+	*Rules
+}
```

Comment:
Not sure if there's a better way to do this implementation or if it's necessarily the correct approach

Reply:
I would not recommend doing it this way.